core/frontend/src/components/parameter-editor/ParameterEditor.vue

```diff
@@ -384,7 +384,7 @@ export default Vue.extend({
         return
       }
       // eslint-disable-next-line no-bitwise
-      for (let v = 0; v < param.value; v += 1) {
```

Comment:
What ?
if `v = 3`:

(1 << v) will be 8
param.value can be any number bigger or smaller than 8

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence

Boolean operations are usually the last one evaluated on a programming language, besides that, there is also the right hand rule for operation evaluation. 
FII, the parentheses are not allowed by the eslint since is a well defined order.

Reply:
yep, you're right, just tested here to be sure :)